WORKSPACE

```diff
@@ -641,7 +636,7 @@ load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "yarn_install
 
 node_repositories(
     package_json = ["//:package.json"],
-    vendored_node = "@nodejs_dev_env",
+    vendored_node = "@nodejs_dev_env" if is_windows else "@node_nix",
```

Comment:
Might be useful to add a comment on why we don’t go via `dev_env` on unix. Do you know why exactly this is causing problems or is this just a case of trial and error and nixpkgs seems to avoid this?

Reply:
I'm not exactly sure what caused the latest failure. However, `dev_env_tool` introduces an additional layer of indirection through symbolic links. We've had a couple issues with that in the past, e.g. https://github.com/digital-asset/daml/issues/2959. This seems to be more problematic for `node_repositories` than in other use-cases of `dev_env_tool`. So, removing that layer of indirection may well avoid the issue.